### DISCUSSION TITLE
DBeaver only able to open single one SQL Editor

### DISCUSSION THREAD

**Discussion by ngnurines** [dbeaver/dbeaver#9316] [Q&A] [answered]
#### System information: 
- Operating system (distribution) and version: Mac OS Catalina 10.15.6
- DBeaver version 7.1.3

#### Connection specification:
- MariaDB 10.3.12
- Google Cloud SQL 5.7.25

#### Describe your question:
- I've tried to check the option Open Separate Connection for each editor
- but it only appear display single editor
- this happens when I click recent editor in database navigator, the sql editor switch to active connection, istead of open other sql editor
- however the windows version is able to open multiple editor, please help since I can't find relate promble from this issue

![image](https://user-images.githubusercontent.com/7721661/87896077-67371600-ca71-11ea-9375-1d6bc531da3c.png)

**Comment by kseniiaguzeeva** [comment, marked as answer]
You can open multiple SQL Editor windows in DBeaver (using New SQL Editor menu, or toolbar, or a database context menu in DB Navigator, or shortcuts).

If I understand you correctly, you have an opened SQL Editor for a database, then press another database in DB Navigator. As a result, the database and schema are changed in SQL Editor window. In this case please disable option **Auto-sync connection with Navigator** (SQL Editor> Auto-sync connection with Navigator)
![dbeaver_XjL5B7RuZg](https://user-images.githubusercontent.com/51405061/87914768-d4e74000-ca79-11ea-9b70-c592540ef7a8.png)

**Open separate connection for each editor** has another meaning. It refers actual physical connection affected by an event. You can see this information in Query manager (Window>Show view>Query manager)
![dbeaver_DHbMT8ILnO](https://user-images.githubusercontent.com/51405061/87915115-5b9c1d00-ca7a-11ea-8353-47685fcf9152.png)
